dlp/src/main/java/dlp/snippets/JobsList.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dlp.snippets;
+
+// [START dlp_list_jobs]
+
+import com.google.cloud.dlp.v2.DlpServiceClient;
+import com.google.privacy.dlp.v2.DlpJob;
+import com.google.privacy.dlp.v2.DlpJobType;
+import com.google.privacy.dlp.v2.ListDlpJobsRequest;
+import com.google.privacy.dlp.v2.ProjectName;
+
+import java.io.IOException;
+
+public class JobsList {
+
+    public static void listJobs() throws IOException {
+        // TODO(developer): Replace these variables before running the sample.
+        // For more info on filters and job types,
+        // see https://cloud.google.com/dlp/docs/reference/rest/v2/projects.dlpJobs/list
+        String projectId = "your-project-id";
+        String filter = "state=DONE";
+        String jobType = "INSPECT_JOB";
```

Comment:
Same here - please move `filter` and `jobType` to the sample

Reply:
inlined these in the sample in 66b34ca